src/Tools/Extensions.ApiDescription.Client/test/TestProjects/files/NSwag.json

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
The JSON files in this directory aren't particularly interesting. Suggest just marking them as Viewed.